pkg/policy/api/rule_validation.go

```diff
@@ -230,6 +241,10 @@ func (e *EgressRule) sanitize() error {
 		}
 	}
 
+	if len(e.ICMPs) > 0 && len(e.ToPorts) > 0 {
+		return fmt.Errorf("The ICMPs block must only be present without any other rules. Define a separate rule to allow other kinds of traffic.")
```

Comment:
Ah, I didn't realize that only `ToPorts` would be relevant here. In that case, we could probably make this msg a bit more specific.

```
The ICMPs block may only be present without ToPorts. Define a separate rule to use ToPorts.
```